specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2019-06-15/bms.json

```diff
@@ -746,7 +746,7 @@
         "tags": [
           "ProtectionPolicies"
         ],
-        "description": "Creates or modifies a backup policy. This is an asynchronous operation. Status of the operation can be fetched\r\nusing GetPolicyOperationResult API.",
```

Comment:
Can you undo these changes? We can undo changes that are not part of DS Move work

Reply:
Done

> Can you undo these changes? We can undo changes that are not part of DS Move work